slices/findd_test.go

```diff
@@ -0,0 +1,78 @@
+package slices
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestFindFirstConsecutiveDigits(t *testing.T) {
+
+	matchDigits := func(t testing.TB, got, want []int) {
+		t.Helper()
+		if !reflect.DeepEqual(got, want) {
```

Comment:
I think you are gonna love https://github.com/stretchr/testify to get nice asserts

Reply:
Javi, good stuff! I'm trying it already -> [PR for structs](https://github.com/rjar2020/learn-go-with-tests/pull/8)

A refactor is coming,  also gonna try [gorc](https://github.com/stretchr/gorc) <3